<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COptionSetting</class>
 <widget class="QDialog" name="COptionSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>設定</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>182</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="adminCheckBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>管理者モード</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="updateCheckBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>証明書更新</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="saveDirGridLayout">
    <item row="0" column="2">
     <widget class="QToolButton" name="saveDirToolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="saveDirLabel">
      <property name="text">
       <string>保存場所：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="saveDirLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="certFileGridLayout">
    <item row="0" column="2">
     <widget class="QToolButton" name="certFileToolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="certFileLabel">
      <property name="text">
       <string>　証明書：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="certFileLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>136</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="saveFmtGridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="saveFmtLabel">
      <property name="text">
       <string>保存形式：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="saveFmtComboBox">
      <item>
       <property name="text">
        <string>jpeg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>png</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>adminCheckBox</zorder>
  <zorder>updateCheckBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
